REFERENCE.md

```diff
@@ -4657,7 +4657,7 @@ The following parameters are available in the `apache::mod::proxy` class.
 
 ##### `proxy_requests`
 
-Data type: `Any`
+Data type: `Boolean`
```

Comment:
This looks like the wrong place. Note you can regenerate this file using `bundle exec rake strings:generate:reference`

Reply:
@aukesj please remove this duplicate section